app/views/admin/schools/show.html.erb

```diff
@@ -0,0 +1,35 @@
+<% content_for :title, "School detail: overview" %>
+
+<h1 class="govuk-heading-xl"><%= @school.name %></h1>
+<%= render partial: "navigation" %>
+<h2 class="govuk-heading-l">Overview</h2>
+<table class="govuk-table">
+  <tbody class="govuk-table__body">
+    <tr class="govuk-table__row">
+      <td class="govuk-table__cell">Induction tutor</td>
+      <td class="govuk-table__cell">
+        <% if @induction_coordinator %>
+          <%= @induction_coordinator.full_name %><br>
+          <%= mail_to @induction_coordinator.email, @induction_coordinator.email  %>
+        <% end %>
+      </td>
+      <td class="govuk-table__cell">
+        <% if @induction_coordinator %>
+        <%= govuk_link_to "Change", edit_admin_induction_coordinator_path(@induction_coordinator) %>
```

Comment:
I don't think this should link here. We're not implementing this bit quite yet, but the spec for changing induction tutor is creating a new account (retaining the old one for audit purposes). Let's leave it as a dead link for now?